content/articles/creating-serverless-functions-faas-with-aws-lambda-and-golang/index.md

```diff
@@ -0,0 +1,292 @@
+Creating Serverless Functions (Faas) with AWS Lambda and Golang
+
+Serverless architecture is a new paradigm that is being used in the development of modern applications. The term "serverless" refers to the abstraction of "servers" from application development. Function-as-a-Service (FaaS) is a serverless method of running modular code on the cloud. You can use FaaS to write code that is triggered by events. For example, when a user clicks on a button in your web application, a specific cloud function is executed.
```

Comment:
Apart from the introduction, I would suggest you to add a separate sub-topics that focuses on the working on serverless architecture, and FaaS.

Reply:
In the introduction, you could talk about:
- The existing systems (with server)
- How server less computing is better than with server computing?
- Give a few live examples that uses serverless architecture

Reply:
I was thinking having a separate sub-topic that focuses entirely on serverless architecture will be unnecessary in the context of this article. The overall purpose of the article is to show readers how to upload their functions to AWS lambda. The process is quite obscured and unclear in AWS docs.

Reply:
@Bamimore-Tomi 

Agreed.

You can do that for sure. But, for the reader, it would be great if you could provide a gist or overall idea.

Reply:
"The existing systems (with server)"

Please I don't understand this point.

"Give a few live examples that uses serverless architecture"
Please I am also not really clear on this.

Thank you.